### ISSUE TITLE
Extension Tutorial is incomplete

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1624]
When trying to follow the tutorial for creating a custom extension (http://sphinx-doc.org/extdev/tutorial.html), the given code fails with the message:

```
ad = make_admonition(todo, self.name, [_('Todo')], self.options,
NameError: global name '_' is not defined
```

After tracking down the todo extension included with Sphinx, I figured out that this can be fixed by adding:

```
#!python

from sphinx.locale import _
```

to the todo python file.

I suggest adding the import _ statement to the tutorial so that it will work as is.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1624
- Originally reported by: [dkeeney_fastacash](https://bitbucket.org/dkeeney_fastacash)
- Originally created at: 2014-11-13T04:00:52.646

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-11-26 11:15:22+00:00_

fix documentation for extension development tutorial. Closes #1624. Thanks to dkeeney_fastacash.

→ <<cset 6993a7254c9c59a3bb4374dd7515dfabd351410d>>

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-11-26 11:16:06+00:00_

[dkeeney_fastacash](https://bitbucket.org/dkeeney_fastacash) Thanks!
